src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderDamageObligationResponse.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.hmpps.prison.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Builder
```

Comment:
would be better with a constructor, can't see the point of a builder with one argument